dhctl/pkg/operations/destroy/destroy.go

```diff
@@ -113,10 +144,67 @@ func (d *ClusterDestroyer) DestroyCluster(autoApprove bool) error {
 	// Stop proxy because we have already got all info from kubernetes-api
 	d.d8Destroyer.StopProxy()
 
-	if err := d.clusterInfra.DestroyCluster(autoApprove); err != nil {
+	if err := infraDestroyer.DestroyCluster(autoApprove); err != nil {
 		return err
 	}
 
 	d.state.Clean()
 	return d.PhasedExecutionContext.Complete()
 }
+
+type staticMastersDestroyer struct {
+	SSHClient *ssh.Client
+}
+
+func newStaticMastersDestroyer(c *ssh.Client) *staticMastersDestroyer {
+	return &staticMastersDestroyer{
+		SSHClient: c,
+	}
+}
+
+func (d *staticMastersDestroyer) DestroyCluster(autoApprove bool) error {
+	if !autoApprove {
+		if !input.NewConfirmation().WithMessage("Do you really want to cleanup control-plane nodes?").Ask() {
+			return fmt.Errorf("Cleanup master nodes disallow")
+		}
+	}
+
+	mastersHosts := d.SSHClient.Settings.AvailableHosts()
+	stdOutErrHandler := func(l string) {
+		log.WarnLn(l)
+	}
+
+	cmd := "test -f /var/lib/bashible/cleanup_static_node.sh || exit 0 && bash /var/lib/bashible/cleanup_static_node.sh --yes-i-am-sane-and-i-understand-what-i-am-doing"
```

Comment:
Why do you ignore missing `cleanup_static_node.sh` here? Shouldn't it be considered a problem?

Reply:
check var/lib/bashible

Reply:
it is unnecessary